src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/EC/ECKeyFileTests.LimitedPrivate.cs

```diff
@@ -102,7 +102,7 @@ public void ReadWriteNistP256ECPrivateKey_LimitedPrivateKey()
                 EccTestData.GetNistP256ReferenceKey());
         }
 
-        [Fact]
+        [ConditionalFact(nameof(CanDeriveNewPublicKey))]
```

Comment:
I think these tests were always designed to be `Fact`. The assertion helpers take a "do we expect this to work?" parameter. So II think this would go back to `Fact` and (as an example in this test):

```csharp
        public void ReadWriteNistP256ExplicitECPrivateKey_LimitedPrivate()
         {
             ReadWriteBase64ECPrivateKey(
                @"
MIIBIgIBAQQgcKEsLbFoRe1W/2jPwhpHKz8E19aFG/Y0ny19WzRSs4qggfowgfcC
AQEwLAYHKoZIzj0BAQIhAP////8AAAABAAAAAAAAAAAAAAAA////////////////
MFsEIP////8AAAABAAAAAAAAAAAAAAAA///////////////8BCBaxjXYqjqT57Pr
vVV2mIa8ZR0GsMxTsPY7zjw+J9JgSwMVAMSdNgiG5wSTamZ44ROdJreBn36QBEEE
axfR8uEsQkf4vOblY6RA8ncDfYEt6zOg9KE5RdiYwpZP40Li/hp/m47n60p8D54W
K84zV2sxXs7LtkBoN79R9QIhAP////8AAAAA//////////+85vqtpxeehPO5ysL8
YyVRAgEB",
                EccTestData.GetNistP256ReferenceKeyExplicit(),
                SupportsExplicitCurves && CanDeriveNewPublicKey);
```

You're basically doing the opposite of this PR's test changes: https://github.com/dotnet/runtime/pull/34654/files#diff-e4a2860c925912f2534407209ef3acb67b86e679156b1c525ad01bcefae0275a

Reply:
I'll take a look at making these go down the "not supported" path in the tests.

Reply:
@jkoritzinsky 👍 For what it's worth, "LimitedPrivate" is this scenario. I probably could have come with a better name, but, if you run in to this term anywhere else, "LimitedPrivate = Can reconstruct public key from private key"

Reply:
That's what I thought, but I wasn't positive. Thanks for clarifying!